dao/src/main/resources/dao.properties

```diff
@@ -0,0 +1,11 @@
+#Database connection
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
I think we can place this in application properties, this is main DB setting and in real production app you will need to provide different settings per environment which application properties supports in spring boot using Spring Profiles